devel/ace/Portfile

```diff
@@ -72,9 +77,6 @@ foreach {key value} [array get os.names]  {
 set subdirs         { ace ACEXML ASNMP Kokyu apps netsvcs }
 
 post-patch {
-    reinplace "s|@MACOSX@|${os.name}|g" \
-        ${worksrcpath}/ace/config.h \
-        ${worksrcpath}/include/makeinclude/platform_macros.GNU
```

Comment:
Since the `os.name` variable is now unused in both ports, can the code that sets that variable be removed from both ports?